index.html

```diff
@@ -403,6 +433,46 @@ <h2>Dictionary {{RTCRtpEncodingParameters}} Members</h2>
       </dl>
     </section>
   </section>
+  <section id="rtcresolutionrestriction">
+    <h3>
+      The {{RTCResolutionRestriction}} dictionary.
+    </h3>
+    <pre class="idl">dictionary RTCResolutionRestriction {
+    unsigned long maxWidth;
+    unsigned long maxHeight;
+};</pre>
+    <section id="rtcresolutionrestriction-attributes">
+      <h2>Dictionary {{RTCResolutionRestriction}} Members</h2>
+      <dl data-link-for="RTCResolutionRestriction" data-dfn-for="RTCResolutionRestriction" class="dictionary-members">
+        <dt>
+          <dfn data-idl>maxWidth</dfn> of type <span class="idlMemberType">unsigned long</span>
+        </dt>
+        <dd>
+          <p>The maximum width that frames will be encoded with without getting
+          downscaled. The restrictions are orientation agnostic, see note below.
+          When scaling is applied, both dimensions of the frame are downscaled
+          using the same factor.</p>
+        </dd>
+        <dt>
+          <dfn data-idl>maxHeight</dfn> of type <span class="idlMemberType">unsigned long</span>
+        </dt>
+        <dd>
+          <p>The maximum height that frames will be encoded with without getting
+          downscaled. The restrictions are orientation agnostic, see note below.
+          When scaling is applied, both dimensions of the frame are downscaled
+          using the same factor.</p>
+        </dd>
+        <p class="note">
+          The restrictions being orientation agnostic means that they will
+          automatically be adjusted to the orientation of the frame being
+          restricted (portrait mode or landscape mode) by swapping width and
+          height if necessary. This means that it does not matter if 1280x720 or
+          720x1280 is specified, both always result in the exact same scaling
+          factor regardless of the orientation of the frame.
```

Comment:
I confirmed that we need to do this and that the C++ implementation already does this.